src/third_party/dartino/object_memory.cc

```diff
@@ -295,7 +302,7 @@ void ObjectMemory::set_up() {
   spare_chunk_ = allocate_chunk(null, TOIT_PAGE_SIZE);
   if (!spare_chunk_) FATAL("Can't allocate initial spare chunk");
   if (spare_chunk_mutex_) FATAL("Can't call ObjectMemory::set_up twice");
```

Comment:
We should probably check that we can allocate the spare chunk mutex. It feels a bit weird that it is on the mutex we check that we cannot call set_up twice.